src/lib/services/access-service.test.ts

```diff
@@ -265,3 +266,19 @@ test('throws error when trying to delete a project role in use by group', async
         );
     }
 });
+
+describe('addAccessToProject', () => {
+    test('should throw an error when you try add access with an empty list of roles', () => {
+        const { accessService } = getSetup();
+        expect(
```

Comment:
we don't await the whole expect

Reply:
Yes! Should be handled in 0408cd10c3eb6f5a9f9fa2ad367692ab0b60c7c2